deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/circuitbreaker/CircuitBreakerConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkiverse.openapi.generator.deployment.circuitbreaker;
+
+import java.util.List;
+
+public final class CircuitBreakerConfiguration {
+
+    private final List<CircuitBreakerClassConfiguration> classConfigurations;
+
+    public CircuitBreakerConfiguration(List<CircuitBreakerClassConfiguration> classConfigurations) {
```

Comment:
If using a map, the constructor should be changed accordingly. 
In any case, you do need to call copyOf